v2/lint/source.go

```diff
@@ -71,6 +72,8 @@ func (s *LintSource) FromString(src string) {
 		*s = RFC5480
 	case RFC5891:
 		*s = RFC5891
+	case CABFBaselineRequirementsOverbearing:
```

Comment:
I know we kind of already had this conversation but I'm curious on @sleevi thoughts on this. We did discuss that this is an actual CABF requirement, right? Are we considering it Overbearing because we can't check that it's actually violating the requirement currently?

Reply:
That was my understanding: That the current design was accounting for a lack of the full chain (see #428 ), and so was using this category as a way of letting CAs distinguish so they can possibly suppress this case. I'm not entirely sold (e.g. they could suppress this lint for certificates issued from their S/MIME CAs, without needing the extra source), and from the spec side, it's still a responder.

Reply:
My main reason to introduce this new category was, to execute this one test for all certificates, regardless if it is a server auth certificate or not. The first option I explored was to keep it in the group `CABFBaselineRequiremens` and remove the filter, that lints from this group only apply to server auth certificates. But to do so, I had to adapt 50+ lints. So I introduced this new group of lints.

Reply:
How do we get this lint through the door now? Currently, I see the following options:
1. We wait until #428 is implemented and finalize this lint again
2. We create a new category for TLS BRG lints that are 'overbearing' (the current approach)
3. We remove the filter for TLS certificates in the base.go file and update most of the over lints as I did here: https://github.com/zmap/zlint/pull/472/commits/5efa6d1e2f7cdc1db39b998baf92e77c522330b4
4. We do some special case handling for this lint in the base.go file as I did here: https://github.com/zmap/zlint/pull/472#pullrequestreview-489707702

Which version do you prefer @cpu @cardonator and @christopher-henderson ?

Reply:
I don't think I'm comfortable with (2) or (4) from a codebase perspective. Creating a new category (2) will cause confusion --- it's not obvious what folks should do about the results that come from it, and (4) inappropriately mixes lint code into the framework. It would be good to get @sleevi's take on this as well since he knows the bug at hand better than I do.

Reply:
For me it is okay to go with (3) but I had the feeling in the discussion of the original PR, that the maintainers weren't happy with adjusting practically all BRG lints - which would be the consequence of choosing (3).